src/basic/path-util.c

```diff
@@ -782,6 +782,10 @@ static int executable_is_good(const char *executable) {
                                "/dev/null");
 }
 
+int fsck_available(void) {
+        return executable_is_good("fsck");
+}
+
 int fsck_exists(const char *fstype) {
```

Comment:
hmm, I find the naming a bit confusing now. `fsck_available()` and `fsck_exists()` is just confusing. Maybe rename the latter `fsck_exists_for_fstype()` and the the former `fsck_exists()`.

Also, I am pretty sure `fsck_exists_for_fstype()` should internally first check `fsck_exists()` too, because we always call fsck through the wrapper, hence if the fstype-specific one exists but the wrapper does not things won't work.